python/tvm/relay/frontend/onnx.py

```diff
@@ -2349,30 +2349,18 @@ class LogSoftmax(OnnxOpConverter):
     """Operator converter for Softmax."""
 
     @classmethod
-    def run_calculation(cls, x, axes):
+    def run_calculation(cls, inputs, attr, params, opset):
         """Run the calculation for Log Softmax calculation."""
-        m = _op.max(x, axes, keepdims=True)
-        e = _op.exp(x - m)
-        s = _op.sum(e, axes, keepdims=True)
-        return x - m - _op.log(s)
```

Comment:
I believe the reason it is done this way is numerical precision concerns. I don't know how often res will underflow/overflow but I would lean toward correctness rather than speed here. This mathematics will likely be fused later on anyway I think

Reply:
Your concern is right, but the underlying computation of the softmax operator already takes the overflow into account.
The code is as below: 
https://github.com/apache/tvm/blob/a3a4155943cd1a8ced35060902907cde2ba44cd8/include/tvm/topi/nn/softmax.h#L86-L114

Reply:
Ah cool